### ISSUE TITLE
Could you provide some detail information on training process?

### ISSUE THREAD

**Issue by ldihao** [HzFu/DENet_GlaucomaScreen#6]
I've been recently implemented your paper using pytorch. But during training, I cannot find the batch size and the strategy to adjust learning rate in your paper. Could you provide those information?
